torch/csrc/api/include/torch/nn/cloneable.h

```diff
@@ -42,7 +42,7 @@ class Cloneable : public virtual Module {
     copy->children_.clear();
     copy->reset();
     TORCH_CHECK(
-        copy->parameters_.size() == parameters_.size(),
+        copy->parameters_.size() == this->parameters_.size(),
```

Comment:
comment on all these places why you did it or people might accidentally remove it.